extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -9,11 +9,20 @@ $color-borders: #CECECE;
 
 @import 'extensions/wikia/ThemeDesigner/css/ThemeTab';
 
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/_colors.scss';
+
+$normal-spacing: 24px;
+$cnw-size: 700px;
+
+.skin-oasis.page-Special_CreateNewWiki {
```

Comment:
These classes are auto-generated. Maybe we can use for example `.WikiaSiteWrapper` here?